### ISSUE TITLE
Quota Config configuration that worked

### ISSUE THREAD

**Issue by venkatesulua1990** [aitalian/apigeex-challenge-lab#2]
Hi ,

Below configuration will work and pass the lab

><Quota continueOnError="false" enabled="true" name="Q-EnforceQuota" type="calendar">
    <Identifier ref="client_id"/>
    <UseQuotaConfigInAPIProduct stepName="VAK-VerifyKey">
        <DefaultConfig>
            <Allow>5</Allow>
            <Interval>1</Interval>
            <TimeUnit>hour</TimeUnit>
        </DefaultConfig>
    </UseQuotaConfigInAPIProduct>
    <Distributed>true</Distributed>
    <Synchronous>true</Synchronous>
    <StartTime>2021-01-01 00:00:00</StartTime>
</Quota>

**Comment by milanmp** [comment]
indeed, just simple values will do, like in [GSP843 Publishing APIs](https://www.cloudskillsboost.google/focuses/32175?parent=catalog#step8)
e.g.

> <Quota continueOnError="false" enabled="true" name="Q-EnforceQuota" type="calendar">
    <DisplayName>Q-EnforceQuota</DisplayName>
    <Distributed>true</Distributed>
    <Synchronous>true</Synchronous>
    <StartTime>2022-8-26 12:00:00</StartTime>
    <UseQuotaConfigInAPIProduct stepName="VAK-VerifyKey">
        <DefaultConfig>
            <Allow>5</Allow>
            <Interval>1</Interval>
            <TimeUnit>hour</TimeUnit>
        </DefaultConfig>
    </UseQuotaConfigInAPIProduct>
</Quota>
